### PR TITLE
change detekt for implement the ArtApp (https://github.com/Architect-…

### PR DESCRIPTION
**Pull request by jamacado**
**Detekt command is not working in Windows** issue #22 

- change detekt for implement the ArtApp [https://github.com/Architect-Coders/ArtApp](url)

- change class CharacterDetailUIModel and CharactersDetailViewModel because detekt report for ruleset

**Testing Done**

On Emulator: Pixel 2XL Api Q

Android Studio 3.5.3
Build #AI-191.8026.42.35.6010548, built on November 15, 2019
JRE: 1.8.0_202-release-1483-b03 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0

**How to test**

In termial the android studio write command **gradlew detekt**

detekt

detektOk

Signed off by:
javier cardona (javier.mcardona@gmail.com)

### CODE REVIEW

File: detekt.gradle [hunk 1]
```diff
@@ -1,19 +1,8 @@
...
-    def input = "$projectDir"
-    def config = "$rootDir/default-detekt-config.yml"
-    def exclude = ".*test.*,.*/resources/.*,.*/tmp/.*"
-    def params = ['-i', input, '-c', config, '-ex', exclude, '-r', '--fail-fast']
-    args(params)
+detekt {
+    config = files("$rootDir/default-detekt-config.yml")
+     //  filters = ".*build.*,.*/resources/.*,.*/tmp/.*"
+    //Optional baseline, uncomment & run gradle command detektBaseline to exclude existing issues
+    //  baseline = file("detekt-baseline.xml")
```
Comment (aliceresponde) [inline detekt.gradle:7]: if the task is a comment .. How can It work? 
I change the format, names and some code style and the task did not identify it. 
Please update it, and explain to us how can I see that the task is working.

File: default-detekt-config.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 build:
-  maxIssues: 1
+  maxIssues: 200
```
Comment (recaldev) [inline default-detekt-config.yml:2]: Why 200? I think max issues should be 1.

File: default-detekt-config.yml [hunk 3]
```diff
@@ -202,7 +202,7 @@ formatting:
     continuationIndentSize: 4
   MaximumLineLength:
     active: true
-    maxLineLength: 140
+    maxLineLength: 200
```
Comment (recaldev) [inline default-detekt-config.yml:205]: Keep this at 140, 200 is too much for a single line length.

File: default-detekt-config.yml [hunk 4]
```diff
@@ -218,8 +218,6 @@ formatting:
   NoEmptyClassBody:
     active: true
     autoCorrect: true
-  NoItParamInMultilineLambda:
```
Comment (recaldev) [inline default-detekt-config.yml:221]: Why did you remove this?

File: default-detekt-config.yml [hunk 5]
```diff
@@ -274,9 +272,6 @@ formatting:
   SpacingAroundRangeOperator:
     active: true
     autoCorrect: true
-  SpacingAroundUnaryOperators:
```
Comment (recaldev) [inline default-detekt-config.yml:277]: Why did you remove this?

File: default-detekt-config.yml [hunk 6]
```diff
@@ -465,7 +460,7 @@ style:
     active: false
   MaxLineLength:
     active: true
-    maxLineLength: 140
+    maxLineLength: 200
```
Comment (recaldev) [inline default-detekt-config.yml:463]: Keep this at 140.

File: default-detekt-config.yml [hunk 7]
```diff
@@ -476,7 +471,7 @@ style:
   NestedClassesVisibility:
     active: false
   NewLineAtEndOfFile:
-    active: true
+    active: false
```
Comment (recaldev) [inline default-detekt-config.yml:474]: Put this as true.

File: detekt.gradle (hunk 1 above)
Comment (recaldev) [reply detekt.gradle:7]: @aliceresponde the baseline is used to add "exclusions" for existing code that is hard to change at the moment, for example in a huge project where lint is implemented recently and in class X is hard to make the changes, the baseline will have reference to class X where the lint would fail to let the build pass (in continuous integration) without marking a lint error.

File: detekt.gradle [hunk 8]
```diff
@@ -1,19 +1,8 @@
...
-    main = "io.gitlab.arturbosch.detekt.cli.Main"
-    classpath = configurations.detekt
-    def input = "$projectDir"
-    def config = "$rootDir/default-detekt-config.yml"
-    def exclude = ".*test.*,.*/resources/.*,.*/tmp/.*"
-    def params = ['-i', input, '-c', config, '-ex', exclude, '-r', '--fail-fast']
-    args(params)
+detekt {
+    config = files("$rootDir/default-detekt-config.yml")
+     //  filters = ".*build.*,.*/resources/.*,.*/tmp/.*"
```
Comment (recaldev) [inline detekt.gradle:5]: Remove this since is not used anymore.

File: detekt.gradle [hunk 9]
```diff
@@ -1,19 +1,8 @@
...
+    //Optional baseline, uncomment & run gradle command detektBaseline to exclude existing issues
```
Comment (recaldev) [inline detekt.gradle:6]: I think we can remove this since we won't use baseline.

File: presentation/src/main/java/com/architect/coders/mu8/characters/detail/CharactersDetailUiModel.kt [hunk 10]
```diff
@@ -1,3 +1,3 @@
 package com.architect.coders.mu8.characters.detail
 
-sealed class CharactersDetailUiModel
\ No newline at end of file
+sealed class CharactersDetailUiModel
```
Comment (recaldev) [inline presentation/src/main/java/com/architect/coders/mu8/characters/detail/CharactersDetailUiModel.kt:3]: What change was done here?

File: presentation/src/main/java/com/architect/coders/mu8/characters/detail/CharactersDetailViewModel.kt [hunk 11]
```diff
@@ -4,11 +4,11 @@ import androidx.lifecycle.LiveData
...
+class CharactersDetailViewModel : ViewModel() {
 
     private val _model = MutableLiveData<CharactersDetailUiModel>()
     val model: LiveData<CharactersDetailUiModel>
         get() {
             return _model
         }
-}
\ No newline at end of file
+}
```
Comment (recaldev) [inline presentation/src/main/java/com/architect/coders/mu8/characters/detail/CharactersDetailViewModel.kt:14]: What change was done here?

**Review by recaldev — CHANGES_REQUESTED** [review]

File: presentation/src/main/java/com/architect/coders/mu8/characters/detail/CharactersDetailUiModel.kt (hunk 10 above)
Comment (jamacado) [reply presentation/src/main/java/com/architect/coders/mu8/characters/detail/CharactersDetailUiModel.kt:3]: sorry i have detekt automatic and i don't realize the change. I'm going to put it back the way it was.

File: presentation/src/main/java/com/architect/coders/mu8/characters/detail/CharactersDetailViewModel.kt (hunk 11 above)
Comment (jamacado) [reply presentation/src/main/java/com/architect/coders/mu8/characters/detail/CharactersDetailViewModel.kt:14]: sorry i have detekt automatic and i don't realize the change. I'm going to put it back the way it was.

**Comment by recaldev** [comment]
We're not going to focus on fixing detekt at the moment.
